src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+function RegistrationForm(props) {
+  const [state, setState] = useState({
+    email: "",
+    password: "",
+    confirmPassword: "",
+  });
+  const handleChange = (e) => {
+    const { id, value } = e.target;
+    setState((prevState) => ({
+      ...prevState,
+      [id]: value,
+    }));
+  };
+  const handleSubmitClick = (e) => {
+    e.preventDefault();
+    if (state.password === state.confirmPassword) {
+      console.log("email: ", state.email, "password: ", state.password);
+    } else {
+      console.log("Passwords do not match!");
+    }
+  };
+  return (
+    <div className="card col-12 col-lg-4 login-card mt-2 hv-center">
+      <form>
+        <div className="form-group text-left">
+          <label htmlFor="exampleInputEmail1">Email Address</label>
+          <input
+            type="email"
+            className="form-control"
+            id="email"
+            placeholder="Enter email"
+            value={state.email}
+            onChange={handleChange}
+          ></input>
+        </div>
+        <div className="form-group text-left">
+          <label htmlFor="exampleInputPassword1">Password</label>
+          <input
+            type="password"
+            className="form-control"
+            id="password"
+            placeholder="Password"
+            value={state.password}
+            onChange={handleChange}
+          ></input>
+        </div>
+        <div className="form-group text-left">
+          <label htmlFor="exampleInputPassword1">Password</label>
+          <input
+            type="password"
+            className="form-control"
+            id="confirmPassword"
```

Comment:
do we need the id here?

Reply:
No we don't

Reply:
Yes, I'm using the Ids for handleChange event.

Reply:
we should use name instead of id. we should never use id in our application unless we really have to since "id" is unique in the whole application and you cant have 2 of the same id. It could be a easy mistake to duplicate the name in another component. 

Instead of id, please use "name" which is the more standard way of reading values in form in HTML and its easier for screen readers to read this as well.